MIGRATION.md

```diff
@@ -0,0 +1,9 @@
+# Migration
+
+## From version 0.15.x to 0.16.0
+
+The re-exports of Puppeteer devices is removed and can be imported from Puppeteer directly.
+
+```ts
+import { KnownDevices } from 'puppeteer';
+```
```

Comment:
I think this migration file needs to be in the package impacted by the change, not in the root

Reply:
@bashmish Good point, I deleted the MIGRATION.md in the root and created one in the relevant packages